### PR TITLE
SUP-17445: Fix loading of styles for custom tag editors

### PR DESCRIPTION
**Pull request by guggi**
Issue: https://jira-gentics.atlassian.net/browse/SUP-17445

### CODE REVIEW

File: cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.scss [hunk 1]
```diff
@@ -42,45 +42,6 @@
...
-                border-bottom-left-radius: 3px;
-                border-left: 1px solid black;
-                border-bottom: 1px solid black;
-                user-select: none;
-
-                &:hover {
-                    opacity: 1.0;
-                }
-            }
-        }
```
Comment (guggi) [inline cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.scss:83]: keep this

File: cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.scss [hunk 2]
```diff
@@ -114,6 +75,14 @@
...
     }
 
+    iframe.master-frame {
+        position: absolute;
+        display: block;
+        width: 100%;
+        height: 100%;
+        border: none;
+    }
+
```
Comment (guggi) [inline cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.scss:85]: remove this

File: cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.ts [hunk 3]
```diff
@@ -106,6 +106,17 @@ import { CombinedPropertiesEditorComponent } from '../combined-properties-editor
...
+
+/**
+ * To make the iframed contentnode pages better fit the look and feel of this app, we apply quite a lot of custom
+ * CSS (see the /custom-styles folder). However, this may have the unwanted effect of overwriting user implementations
+ * (e.g. custom-styled tag fill dialogs). Therefore we will use the value of this constant to control whether or not
+ * our custom styles are being applied or not.
+ *
+ * TODO: actual implementation of styles toggle still needs to be done. This flag is just there in case it is needed.
+ */
+const APPLY_CUSTOM_STYLES = true;
```
Comment (guggi) [inline cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.ts:118]: check if needed

File: cms-ui/apps/editor-ui/src/app/core/providers/resource-url-builder/resource-url-builder.ts [hunk 4]
```diff
@@ -89,6 +89,17 @@ export class ResourceUrlBuilder {
...
+     * Returns the URL of the split-screen language comparison page.
+     */
+    comparePageLanguages(nodeId: number, pageId: number, languageVariantId: number): string {
+        const params = this.createParamsString({
+            otherPageId: languageVariantId,
+            nodeId: nodeId,
+        });
+        return `${API_BASE_URL}/page/diff/${pageId}?${params}`;
+    }
+
```
Comment (guggi) [inline cms-ui/apps/editor-ui/src/app/core/providers/resource-url-builder/resource-url-builder.ts:102]: remove

File: cms-ui/apps/editor-ui/src/app/content-frame/providers/iframe-manager/iframe-manager.service.ts [hunk 5]
```diff
@@ -277,7 +277,9 @@ export class IFrameManager {
                 case EditMode.PREVIEW:
                 case EditMode.EDIT:
                     if (editorState.itemType === 'page') {
-                        if (editorState.editMode === 'preview') {
+                        if (editorState.compareWithId) {
+                            url = this.resourceUrlBuilder.comparePageLanguages(nodeId, itemId, editorState.compareWithId);
+                        } else if (editorState.editMode === 'preview') {
```
Comment (guggi) [inline cms-ui/apps/editor-ui/src/app/content-frame/providers/iframe-manager/iframe-manager.service.ts:282]: remove

**Review by deckdom — APPROVED** [review]

File: cms-ui/apps/editor-ui/src/app/content-frame/providers/iframe-manager/iframe-manager.service.ts (hunk 5 above)
Comment (guggi) [reply cms-ui/apps/editor-ui/src/app/content-frame/providers/iframe-manager/iframe-manager.service.ts]: 4a3057b5

File: cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.ts (hunk 3 above)
Comment (guggi) [reply cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.ts]: 4a3057b5

File: cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.scss (hunk 2 above)
Comment (guggi) [reply cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.scss]: 4a3057b5

File: cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.html [hunk 6]
```diff
@@ -82,26 +82,15 @@
...
-            <iframe
-                #iframe
-                class="frame master-frame"
-                [attr.data-item-type]="currentItem?.type"
-                [attr.data-edit-mode]="editMode"
-                name="master-frame"
-                aria-hidden="false"
-            ></iframe>
-            <div class="frame-language-indicator" [class.hidden]="!comparePageId">{{ itemLanguage?.name }}</div>
-        </div>
```
[the remark this answers is not in the document]
Comment (guggi) [reply cms-ui/apps/editor-ui/src/app/content-frame/components/content-frame/content-frame.component.html]: 4a3057b5

**Pull request merged**